Mail/Components/ThreadCell/ThreadCell.swift

```diff
@@ -77,9 +77,13 @@ struct ThreadCellDataHolder {
         }
     }
 
-    func contactConfiguration(contextMailboxManager: MailboxManager) -> ContactConfiguration {
+    func contactConfiguration(thread: Thread, contextMailboxManager: MailboxManager) -> ContactConfiguration {
```

Comment:
Maybe pass directly bimi instead of the whole thread since no other property is used